packages/breadcrumbs/docs/Breadcrumbs.mdx

```diff
@@ -16,9 +16,9 @@ import { Breadcrumbs } from '@warp-ds/react';
 <Breadcrumbs>
   <a href="#/url/1">Eiendom</a>
   <a href="#/url/2">Bolig til salgs</a>
-  <a href="#/url/3" aria-current="page">
+  <span href="#/url/3" aria-current="page">
```

Comment:
Should this still have the href attribute when converted to a span?

Reply:
Ah, good catch! Fixed in [21deafd](https://github.com/warp-ds/react/pull/23/commits/21deafddb91497c85355dbd407796b21d80b78d6)